cpu/stm32_common/include/periph_cpu_common.h

```diff
@@ -148,6 +148,16 @@ typedef uint32_t gpio_t;
  */
 #define SPI_HWCS(x)         (SPI_HWCS_MASK | x)
 
+/**
+ * @name    Use the shared I2C functions
+ * @{
+ */
+/** Use read regs function from periph common */
+#define PERIPH_I2C_NEED_READ_REG
+/** Use write regs function from periph common */
+#define PERIPH_I2C_NEED_WRITE_REG
```

Comment:
You're not using `PERIPH_I2C_NEED_READ_REGS` and `PERIPH_I2C_NEED_WRITE_REGS` ?

Reply:
because it's a special case of read/write_regs. It's not needed in this driver. Maybe we could also get rid of read/write_regs and use the periph_common implementation but I wasn't able the find a suitable solution.

Reply:
This can be done later after the I2C refactoring then.